src/modules/twap/hooks/useCreateTwapOrder.ts

```diff
@@ -41,11 +44,22 @@ export function useCreateTwapOrder() {
       outputAmount: outputCurrencyAmount,
     }
 
+    const twapFlowAnalyticsContext: SwapFlowAnalyticsContext = {
+      account,
+      recipient: twapOrder.receiver,
+      recipientAddress: twapOrder.receiver,
+      marketLabel: [inputCurrencyAmount.currency.symbol, outputCurrencyAmount.currency.symbol].join(','),
+      // TODO: change to TWAP?
+      orderClass: OrderClass.LIMIT,
+    }
+
     const startTime = Math.round((Date.now() + ms`1m`) / 1000) // Now + 1 min
     const twapOrderWithStartTime = { ...twapOrder, startTime }
     const paramsStruct = buildTwapOrderParamsStruct(chainId, twapOrderWithStartTime)
     const orderId = getConditionalOrderId(paramsStruct)
 
+    // TODO: swap, approveAndPresign or something else?
+    tradeFlowAnalytics.trade(twapFlowAnalyticsContext)
```

Comment:
I think this should be a new type of event. Something like `place twap order`.
Or to be more generic, `place advanced order` and some inner property identifying thin advanced order is a TWAP

Reply:
Added separate event, it will look like this
<img width="661" alt="Screenshot 2023-06-15 at 16 49 13" src="https://github.com/cowprotocol/cowswap/assets/34926005/fe107651-98b0-4a11-bea2-3569b0778b2e">